metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/operators/datahub.py

```diff
@@ -60,4 +66,36 @@ def __init__(  # type: ignore[no-untyped-def]
         self.metadata = mces
 
     def execute(self, context):
+        if context:
+            jinja_env = self.get_template_env()
+            for mce in self.metadata:
+                if isinstance(mce, MetadataChangeEventClass):
+                    for key in mce.keys():
+                        value = getattr(mce, key)
+                        if isinstance(value, DatasetSnapshotClass):
```

Comment:
this is much too specific to dataset snapshots

this should work for MetadataChangeEventClass and MetadataChangeProposalWrapper, regardless of what the contents are